### PR TITLE
Drop container=pod label when scraping container network metrics

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]

File: deploy/helm/prometheus-overrides.yaml [hunk 1]
```diff
@@ -307,6 +307,9 @@ prometheus:
           - action: keep
             regex: kubelet;(?:container_network_receive_bytes_total|container_network_transmit_bytes_total)
             sourceLabels: [job, __name__]
+          - action: drop
```
Comment (frankreno) [inline deploy/helm/prometheus-overrides.yaml:310]: As discussed, we want to use label drop to drop the label `container=POD` from the `container_network_receive_bytes_total` and `container_network_transmit_bytes_total` metrics. Not drop the metrics themselves.

Comment (pmalek-sumo) [reply deploy/helm/prometheus-overrides.yaml:310]: Updated, should be good now

Comment (frankreno) [reply deploy/helm/prometheus-overrides.yaml]: Did you test this locally and confirm that we still get the metrics in Sumo but they no longer have the `container=POD` ? The reason I ask is I looked at the Prometheus relabel docs - https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config

This indicates that when using `labelDrop` action, it matches against a `regex` property. Does not explicitly state it works for `sourceLabels` so just want to make sure.

Comment (pmalek-sumo) [reply deploy/helm/prometheus-overrides.yaml]: So I based that on this https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#relabelconfig specifically on:

> sourceLabels  - The source labels select values from existing labels. Their content is concatenated using the configured separator **and matched against the configured regular expression** for the replace, keep, and drop actions.

at the same time:


> regex - Regular expression against which the extracted value is matched. Default is **'(.*)'**

And as I look at the metrics coming from my KOPS cluster it indeed remove the `cluster` label so I cannot see it anymore (unless I make the timerange wide enough to cover my testing yesterday where I haven't managed to configure the `relabel_config` yet)

Comment (pmalek-sumo) [reply deploy/helm/prometheus-overrides.yaml]: I obviously meant `container` label, not `cluster` label.

**Pull request merged**